examples/README.md

```diff
@@ -0,0 +1,3 @@
+## JetStream Examples
+
+It's a complete implementation of [JetStream Work Queues Example](https://github.com/nats-io/jetstream/blob/master/examples/workqueue/README.md)
```

Comment:
Nice one, should we put the stream.go in the workqueue directory?

I like the example - I was in the text avoiding using jsm.go since there's a perception that its REQUIRED and so any language without an analogue library are left out of luck, but I think it's fine in your example code there.

Reply:
> Nice one, should we put the stream.go in the workqueue directory?

Yeah, we can transfer this `stream.go` to that directory.
As starting jetstream server, creating stream/consumer is part of this repo, that's why I placed it here.

But putting this example into [jetstream](https://github.com/nats-io/jetstream) repo, makes more sense.